automation/include/balena-deploy.inc

```diff
@@ -270,7 +270,7 @@ balena_deploy_to_s3() {
 
 	local _s3_cmd="s4cmd --access-key=${_s3_access_key} --secret-key=${_s3_secret_key}"
 	local _s3_sync_opts="--recursive --API-ACL=${_s3_policy}"
-	docker pull ${_namespace}/resin-img:master
+	docker pull ${_namespace}/balena-img:master
```

Comment:
Are balena-img deployments tagged? Should we fix a tag instead of pulling master to avoid upstream changes breaking OS deployments?

Reply:
@jakogut I proposed fixing a tag so that if a specific OS version works (which fixes balena-yocto-script to a sha1), it never breaks.
My first instinct is that if there is some new feature the OS needs from a newer balena-img, the right thing is to PR and explain the update in the commit.
Not sure what's the actual use of overriding this in Jenkins, do you have a use case in mind?

Reply:
> Not sure what's the actual use of overriding this in Jenkins, do you have a use case in mind?

Nope, that reasoning makes sense to me. I was thinking it's one less thing we need to make PRs for, but your explanation makes sense, I just hadn't thought about it that way. I'll update this with a tag.